provision.sh

```diff
@@ -38,7 +38,7 @@ echo "---------------------------------------------"
 # And that would prevent ~/.bashrc to work properly because \r would be unrecognized
 # Ruby needed for heroku toolbelt
 # notice that this is not a rigorous Ruby install, where we typically use rvm
-apt-get install -y --no-install-recommends heroku-toolbelt build-essential dos2unix python-pip man ruby python-dev python3-dev libpq-dev postgresql postgresql-contrib 
+apt-get install -y --no-install-recommends heroku-toolbelt build-essential dos2unix python-pip man ruby python-dev python3-dev libpq-dev postgresql postgresql-contrib curl
```

Comment:
@raisedadead @alayek please don't remove libpq-dev, this is needed for the proper installation of psycop2 within the virtualenv, otherwise Python will not interface with Postgres and nor will Heroku.